README.md

```diff
@@ -21,9 +25,14 @@ You have two options of how you want to build your UI
 
 ### Testing
 
+1. `yarn test:watch` which watches tests as you change them
+2. `yarn test:ci` which sets the continuous integration environment flag
```

Comment:
A small thing—I recommend removing the word "which" from these two items.

Reply:
Love trimming the fat -- updated!